src/components/Metrics/index.js

```diff
@@ -0,0 +1,94 @@
+import { RichTypography } from "@commons-ui/core";
+import { IconButton, Grid, useMediaQuery } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import DataVisualCard from "@/pesayetu/components/DataVisualCard";
+import Section from "@/pesayetu/components/Section";
+
+function Metrics({ items, title, ...props }) {
+  const classes = useStyles(props);
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.only("md"));
+  if (!items?.length) {
+    return null;
+  }
+  return (
+    <div className={classes.root}>
+      <Section classes={{ root: classes.section }}>
+        <RichTypography variant="h2" className={classes.sectionTitle}>
+          {title}
+        </RichTypography>
+        {items?.map((item, index) => (
+          <Grid
+            key={item.title}
+            container
+            direction={index % 2 === 0 ? "row" : "row-reverse"}
+            justifyContent="space-between"
+            alignItems="flex-start"
+            className={classes.metrics}
+          >
+            <Grid item xs={12} md={5} lg={3}>
+              <Grid container item direction="row" alignItems="center">
```

Comment:
```suggestion
              <Grid container direction="row" alignItems="center">
```
I think even a normal `div` or `Box display="flex" alignItems="center"` would work here...